pages/dao/[symbol]/proposal/components/instructions/UXDProtocol/Mint.tsx

```diff
@@ -0,0 +1,223 @@
+import { useContext, useEffect, useState } from 'react'
+import * as yup from 'yup'
+import {
+  Governance,
+  ProgramAccount,
+  serializeInstructionToBase64,
+} from '@solana/spl-governance'
+
+import { PublicKey, TransactionInstruction } from '@solana/web3.js'
+import { UXD } from '@uxd-protocol/uxd-client'
+import Input from '@components/inputs/Input'
+import Select from '@components/inputs/Select'
+import useGovernanceAssets from '@hooks/useGovernanceAssets'
+import {
+  DEPOSITORY_TYPES,
+  getDepositoryMintSymbols,
+  getDepositoryTypes,
+  mintUXDIx,
+} from '@tools/sdk/uxdProtocol'
+import { checkInitTokenAccount, findATAAddrSync } from '@utils/ataTools'
+import { isFormValid } from '@utils/formValidation'
+import {
+  UiInstruction,
+  UXDMintForm,
+} from '@utils/uiTypes/proposalCreationTypes'
+import { NewProposalContext } from '../../../new'
+import GovernedAccountSelect from '../../GovernedAccountSelect'
+import SelectOptionList from '../../SelectOptionList'
+import useWalletOnePointOh from '@hooks/useWalletOnePointOh'
+import useLegacyConnectionContext from '@hooks/useLegacyConnectionContext'
+
+const schema = yup.object().shape({
+  governedAccount: yup
+    .object()
+    .nullable()
+    .required('Governance account is required'),
+  depositoryType: yup.string().required('Valid Depository type is required'),
+  uxdProgram: yup.string().required('UXD Program address is required'),
+  user: yup.string().required('user is required'),
+  collateralName: yup.string().required('Collateral Name address is required'),
+  collateralAmount: yup
+    .number()
+    .moreThan(0, 'Collateral amount should be more than 0')
+    .required('Collateral Amount is required'),
+})
+
+const Mint = ({
```

Comment:
Notes:

The new `mint` instruction got 29 accounts + data, I don't think current DAO supports it (needs Address Lookup Table I think).

For `mintWithMercurialDepository` and `MintWithCredixDepository`, they now are restricted to the `authority` of the program.